Source/NSScrollView.m

```diff
@@ -1755,6 +1755,40 @@ - (id) initWithCoder: (NSCoder*)aDecoder
   return self;
 }
 
+- (BOOL)automaticallyAdjustsContentInsets
+{
+  // FIXME
+  return NO;
+}
+
+- (void)setAutomaticallyAdjustsContentInsets: (BOOL)adjusts
+{
+  // FIXME
+  return;
+}
+
+- (NSEdgeInsets)contentInsets
+{
+  // FIXME
+  return NSEdgeInsetsZero;
+}
+- (void)setContentInsets:(NSEdgeInsets)edgeInsets
+{
+  // FIXME
+  return;
+}
+
+- (NSEdgeInsets)scrollerInsets
+{
+  // FIXME
+  return NSEdgeInsetsZero;
+}
+- (void)setScrollerInsets:(NSEdgeInsets)insets
+{
+  // FIXME
+  return;
```

Comment:
I agree with the warning but disagree with returning the value that is passed.

While we do not implement this, `Zero` is the right thing to return, because that's the reality -- no matter what we tell it to do, the scroll view does *not* have insets.